kill.go

```diff
@@ -53,16 +53,16 @@ signal to the init process of the "ubuntu01" container:
 	},
 }
 
-func parseSignal(rawSignal string) (unix.Signal, error) {
+func parseSignal(rawSignal string) (syscall.Signal, error) {
```

Comment:
I think we can keep this as `unix.Signal`. Given that the `syscall` package is deprecated / frozen, and this is all Linux-only code (so we don't have to consider (e.g.) Windows; `unix.Signal` is an alias for `syscall.Signal`, so I don't think keeping it should be a problem; https://github.com/golang/sys/blob/751e447fb3d0a97f584890476adddc1d56307388/unix/aliases.go#L13-L14

Reply:
Sure, done!